### PR TITLE
Integrate POSTLogger to the demo app

### PR DESCRIPTION
**Pull request by RidipDe**
*Description of changes*
Integrating the MeetingSessionPOSTLogger into the demo app

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: demos/browser/app/meeting/meeting.ts [hunk 1]
```diff
@@ -550,7 +551,12 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
   }
 
   async initializeMeetingSession(configuration: MeetingSessionConfiguration): Promise<void> {
-    const logger = new ConsoleLogger('SDK', LogLevel.DEBUG);
+    let logger: MeetingSessionPOSTLogger | ConsoleLogger = null;
+    if (!(DemoMeetingApp.BASE_URL.includes('localhost') || DemoMeetingApp.BASE_URL.includes("127.0.0.1"))) {
```
Comment (simmkyu) [inline demos/browser/app/meeting/meeting.ts]: https://github.com/aws/amazon-chime-sdk-js/pull/162 will overrride this `if` statement, but replace ```"``` to ```'``` when you resolve other conflicts.

File: demos/serverless/src/handlers.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const meetingsTableName = process.env.MEETINGS_TABLE_NAME;
 const attendeesTableName = process.env.ATTENDEES_TABLE_NAME;
 const sqsQueueArn = process.env.SQS_QUEUE_ARN;
 const provideQueueArn = process.env.USE_EVENT_BRIDGE === 'false';
+const logGroupName = process.env.BROWSER_LOG_GROUP_NAME
```
Comment (simmkyu) [inline demos/serverless/src/handlers.js]: - missing `;`
- Do not need to do in this PR, but we need to apply ESLint on this file.

File: demos/browser/app/meeting/meeting.ts [hunk 3]
```diff
@@ -550,7 +551,12 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
   }
 
   async initializeMeetingSession(configuration: MeetingSessionConfiguration): Promise<void> {
-    const logger = new ConsoleLogger('SDK', LogLevel.DEBUG);
+    let logger: MeetingSessionPOSTLogger | ConsoleLogger = null;
```
Comment (asnyatkov) [inline demos/browser/app/meeting/meeting.ts]: You should pass the logger type from the server in JoinInfo structure to avoid localhost check below.

Comment (RidipDe) [reply demos/browser/app/meeting/meeting.ts]: PR 162 is making the change

Comment (asnyatkov) [reply demos/browser/app/meeting/meeting.ts]: In that case you should not touch this code. And it also makes deployment safer.

**Comment by simmkyu** [comment]
In this PR, we don't make changes to `meeting.ts`. Will we enable the POSTLogger with #161?

**Comment by RidipDe** [comment]
> In this PR, we don't make changes to `meeting.ts`. Will we enable the POSTLogger with #161?

Yes, with PR 162

**Pull request merged**